js/RTCIceCandidate.js

```diff
@@ -62,102 +60,102 @@ module.exports = RTCIceCandidate;
 
 	var CANDIDATE_TYPE = '';
 	Object.keys(candidateType).forEach(function (key) {
-	  CANDIDATE_TYPE += candidateType[key] + '|';
+		CANDIDATE_TYPE += candidateType[key] + '|';
 	});
 	CANDIDATE_TYPE += TOKEN;
 
 	var pattern = {
-	  COMPONENT_ID: '[0-9]{1,5}',
-	  FOUNDATION: '[a-zA-Z0-9\\+\\/\\-]+',
-	  PRIORITY: '[0-9]{1,10}',
-	  TRANSPORT: transport.UPD + '|' + TOKEN,
-	  CONNECTION_ADDRESS: IPV4ADDR + '|' + IPV6ADDR,
-	  PORT: '[0-9]{1,5}',
-	  CANDIDATE_TYPE: CANDIDATE_TYPE
+		COMPONENT_ID: '[0-9]{1,5}',
+		FOUNDATION: '[a-zA-Z0-9\\+\\/\\-]+',
+		PRIORITY: '[0-9]{1,10}',
+		TRANSPORT: transport.UPD + '|' + TOKEN,
+		CONNECTION_ADDRESS: IPV4ADDR + '|' + IPV6ADDR,
+		PORT: '[0-9]{1,5}',
+		CANDIDATE_TYPE: CANDIDATE_TYPE
 	};
 
 	return function candidateToJson(iceCandidate) {
-	    var iceCandidateJson = null;
-
-	    if (iceCandidate && typeof iceCandidate === 'string') {
-	      var ICE_CANDIDATE_PATTERN = new RegExp(
-	        'candidate:(' + pattern.FOUNDATION + ')' +      // 10
-	        '\\s(' + pattern.COMPONENT_ID + ')' +           // 1
-	        '\\s(' + pattern.TRANSPORT + ')' +              // UDP
-	        '\\s(' + pattern.PRIORITY + ')' +               // 1845494271
-	        '\\s(' + pattern.CONNECTION_ADDRESS + ')' +     // 13.93.107.159
-	        '\\s(' + pattern.PORT + ')' +                   // 53705
-	        '\\s' +
-	        'typ' +
-	        '\\s(' + pattern.CANDIDATE_TYPE + ')' +         // typ prflx
-	        '(?:\\s' +
-	        'raddr' +
-	        '\\s(' + pattern.CONNECTION_ADDRESS + ')' +     // raddr 10.1.221.7
-	        '\\s' +
-	        'rport' +
-	        '\\s(' + pattern.PORT + '))?'                    // rport 54805
-	      );
-
-	      var iceCandidateFields = iceCandidate.match(ICE_CANDIDATE_PATTERN);
-	      if (iceCandidateFields) {
-	        iceCandidateJson = {};
-	        Object.keys(candidateFieldName).forEach(function (key, i) {
-	          // i+1 because match returns the entire match result
-	          // and the parentheses-captured matched results.
-	          if (iceCandidateFields.length > (i + 1) && iceCandidateFields[i + 1]) {
-	            iceCandidateJson[candidateFieldName[key]] = iceCandidateFields[i + 1];
-	          }
-	        });
-	      }
-	    }
-
-	    return iceCandidateJson;
+		var iceCandidateJson = null;
+
+		if (iceCandidate && typeof iceCandidate === 'string') {
+			var ICE_CANDIDATE_PATTERN = new RegExp(
+				`candidate:(${pattern.FOUNDATION})` + // 10
+					`\\s(${pattern.COMPONENT_ID})` + // 1
+					`\\s(${pattern.TRANSPORT})` + // UDP
+					`\\s(${pattern.PRIORITY})` + // 1845494271
+					`\\s(${pattern.CONNECTION_ADDRESS})` + // 13.93.107.159
+					`\\s(${pattern.PORT})` + // 53705
+					'\\s' +
+					'typ' +
+					`\\s(${pattern.CANDIDATE_TYPE})` + // typ prflx
+					'(?:\\s' +
+					'raddr' +
+					`\\s(${pattern.CONNECTION_ADDRESS})` + // raddr 10.1.221.7
+					'\\s' +
+					'rport' +
+					`\\s(${pattern.PORT}))?` // rport 54805
```

Comment:
These are the only functional changes that I made.  Converted to use template strings so that the appropriate groups were kept on the same line.

Reply:
Do this template syntax work on webview in iOS 10+?

Reply:
https://caniuse.com/template-literals - Looks like 9+ supports them